tests/test_azure.py

```diff
@@ -58,37 +62,173 @@ def test_parse_image_name(monkeypatch):
     assert az.parse_image_name('do not match') is None
 
 
-def test_cleanup_sle_images_container_too_many(monkeypatch):
-    test_name = 'quantity'
+def test_cleanup_sle_images_container(monkeypatch):
+    class FakeBlockBlobService:
+        deleted_blobs = list()
+
+        def list_blobs(self, container):
+            last_modified = None
```

Comment:
why ?  it is already `last_modified=None`  in constructor

Reply:
Left over. I think I played with times or so... will clean it up